README.md

```diff
@@ -1 +1,31 @@
-# Login-page
\ No newline at end of file
+<<<<<<< HEAD
+# Login-page
+=======
```

Comment:
You committed the file with Git conflicts. resolve the conflicts and commit again